fix display (#196)
Ostorlab/KB: WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md

> **Content Security Policy** (CSP): is a computer security standard that provides an added layer of protection against Cross-Site Scripting (XSS), clickjacking, and other client-side attacks that rely on executing malicious content in the context of a trusted web resource.

Reviewer: ```suggestion
**Content Security Policy** (CSP): A computer security standard that provides an added layer of protection against Cross-Site Scripting (XSS), clickjacking, and other client-side attacks that rely on executing malicious content in the context of a trusted web resource.
```

Reviewer 2: @Zakaria-Latif, I think this is out of the PR context.

Reviewer: The PR says `fix display`, I thought it was meant to correct typos, grammar, and spelling !!.

You can selectively specify which data sources should be permitted in your web application using appropriate CSP directives in HTTP response headers. This article shows how to use CSP headers to protect websites against XSS attacks and other attempts to bypass the same-origin policy.

CSP can be enabled by instructing the browser with a Content-Security-Policy directive in a response header:

```html
 Content-Security-Policy: script-src 'self';
```

> Or in a meta tag;

Reviewer: ```suggestion
Or in a meta tag:
```

```html
<meta http-equiv="Content-Security-Policy" content="script-src 'self';"> 
```

> You can restrict script loading only to the same domain in the above example. It will also limit inline script executions in the element attributes and the event handlers. 

Reviewer: ```suggestion
You can restrict script loading only to the same domain in the above example. It also limits inline script execution in element attributes and event handlers. 
```

There are various directives that you can use by declaring CSP:

- **script-src**: Restricts the script loading resources to the ones you declared. By default, it disables inline script executions unless you permit the evaluation functions and inline scripts by the unsafe-eval and unsafe-inline keywords.
- **base-uri**: The base element is used to resolve a relative URL to an absolute one. By using this CSP directive, you can define all possible URLs which could be assigned to the base-href attribute of the document. 
- **frame-ancestors**: It is very similar to X-Frame-Options HTTP header. It defines the URLs by which the page can be loaded in an iframe.
- **frame-src / child-src**: frame-src is the deprecated version of child-src. Both define the sources that can be loaded by iframe on the page. (Please note that frame-src was brought back in CSP 3)

> - **object-src** : Defines the resources that can be loaded by embedding such as Flash files, Java Applets.

Reviewer: ```suggestion
- **object-src**: Defines the resources that can be loaded by embedding such as Flash files, Java Applets.
```
